joomla/joomla-cms
# [3.5] Remove com_banners and mod_banners from core new installs

Following the roadmap for Joomla 3.5: 
`Remove additional components and all associated pieces. ([...] com_banners).[...]`
http://developer.joomla.org/cms/roadmap.html

This will remove com_banners from the core new installs.
### how to test (first test)
- download my branche: https://github.com/zero-24/joomla-cms/archive/bannerRemoval.zip
- install as normal joomla 3.4
- make sure com_banners is gone and no errors are there e.g. SQL or others.
### how to test (seccond test)
- install normal 3.4
- update with a update package (needs to be created and uploaded to joomla.org)
- see that nothing is changed and com_banners still work as expeced.
### Open questions
- what we need to do outsited of this PR?
- how can we create a update package and upload it to joomla.org? (for the seccond test)
### More information
- New com_banners repo: https://github.com/joomla-extensions/banners
- First PR: https://github.com/joomla-extensions/banners/pull/1
